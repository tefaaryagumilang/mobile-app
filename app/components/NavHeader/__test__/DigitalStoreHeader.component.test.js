import React from 'react';
import renderer from 'react-test-renderer';
import DigitalStoreHeader from '../DigitalStoreHeader.component';

describe('NavHeader Componenent: DigitalStoreHeader', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <DigitalStoreHeader navigate={jest.fn()} dispatch={jest.fn()}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});