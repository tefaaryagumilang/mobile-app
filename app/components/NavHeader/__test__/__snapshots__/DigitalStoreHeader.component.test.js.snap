// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavHeader Componenent: DigitalStoreHeader renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
    }
  }
>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 20,
          },
          Object {
            "color": "#000000",
            "paddingLeft": 10,
          },
          Object {
            "fontFamily": "simas",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
  </View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 32,
          },
          Object {
            "color": "#000000",
            "paddingLeft": 10,
          },
          Object {
            "fontFamily": "simas",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
    <View />
  </View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 20,
          },
          Object {
            "color": "#000000",
            "paddingLeft": 10,
          },
          Object {
            "fontFamily": "simas",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
    <View />
  </View>
</View>
`;
